README.md

```diff
@@ -33,10 +33,16 @@ NOTE: First set the environment variables listed above.
 
 ## Data scraper
 
-Do not run script in script folder, run it in the main folder!
+Do not run script in scraper folder, run it in the main folder!
```

Comment:
Can we wire this up to a GitHub action so it runs automatically whenever changes to the scraper are pushed to a PR or branch?

Reply:
Will do it